tests/test_sqlite.py

```diff
@@ -1,34 +1,98 @@
-from ms2query.sqlite import create_table_structure, \
-    add_spectra_to_database, get_spectra_from_sqlite
-from ms2query.app_helpers import gather_test_json
 import os
-import json
-from matchms.Spectrum import Spectrum
+import pandas as pd
+import numpy as np
+from matchms import Spectrum
+from ms2query.app_helpers import load_pickled_file
+from ms2query.create_sqlite_database import make_sqlfile_wrapper
+from ms2query.query_from_sqlite_database import \
+    get_tanimoto_score_for_inchikeys, get_spectra_from_sqlite
 
 
-def test_sqlite_functions(tmp_path):
-    """Tests the functions create_table_structure, add_spectra_to_database
-    and get_spectra_from sqlite"""
+def test_sqlite_functions_wrapper(tmp_path):
```

Comment:
I would remove this wrapper entirely and replace it by a helper function and separate tests for the three functions.